index.md

```diff
@@ -2,13 +2,30 @@
 layout: post
 title: About this guide
 permalink: /
-#sidenav: about
+sidenav: about
 sticky_sidenav: true
 ---
-This guide is for people at 18F who are wondering what to expect from a product manager on their team, as well as for product managers and those filling that role to understand what their team expects from them. This guide also serves as a resource for product management best practices at 18F (generalized to apply to any of the specific types of product work we do, like Custom Partner Solutions (CPS) projects, product or platform work, Acquisition projects, consulting, etc.)
+
+18F product managers join cross-functional teams to ensure successful delivery and that user-centered, agile approaches guide product strategy. The 18F Product Guide is a starting point for consulting Product Managers at 18F.
+
+**18F Product Guide** is primarily written for internal product managers, but we hope it may be a useful reference for any product manager working in government. This guide communicates what to expect from a product manager on your team, as well as an understanding of what your team expects from product managers. This guide offers a resource for product management best practices at 18F across multiple phases and types of consulting work.
 
 ## Product Management at 18F
 
-As a product manager at 18F, you may be put in a client-facing role to help a government partner build a mission critical product.  You may also be working on one of TTS’s flagship products, like <a href="https://eregs.github.io/" target="_blank">eRegs</a>, <a href="https://federalist.18f.gov" target="new">Federalist</a>, <a href="https://cloud.gov" target="new">cloud.gov</a> or <a href="https://login.gov/" target="_blank">login.gov</a>. In all cases, you will create or steward the product vision and strategy, and drive a cross-functional team to deliver the right product to the right audience. We will talk more about this in the [Defining the future state]({{site.baseurl}}/define/) section.
+As a PM at 18F, you will find yourself in a client-facing consulting role to help a government partner build or buy a mission-critical product. As we work across the entire federal government and can now support state and local governments, you may encounter subject matter that is entirely new to you. Or, you may be working on one of TTS’ flagship products, like [Federalist](https://federalist.18f.gov/), [cloud.gov](https://cloud.gov/) or [login.gov](https://login.gov/). In all cases, you will develop and steward a product vision and strategy, and drive a cross-functional team to deliver the right product to the right audience. This involves different activities and priorities at different phases of the work—whether you're in discovery or implementation mode—you are creating the space and opportunity for your team to deliver on intended outcomes, rooted in evidence.
```

Comment:
Update from "client-facing" to "partner-facing" and double check content across the guide reflects this as well.

Reply:
"Partner-facing role to help a government partner" feels a little repetitive but I hear you. We have client referenced four places. Should this hold up pushing these changes or could it be something we ask writing lab to look out for as a consistency ask?  (cc Thoughts, @ninamak? I'm sure you have thought about client vs partner while writing the content)

Reply:
I actually did not know we have a preference for partner vs client and had not thought about it before.  I would be interested in getting input from the writing lab on this.  Especially because, in this particular case, I would have thought the use of "client" in "client-facing" feels more appropriate than partner, because I've never heard anyone talk about consulting roles as partner-facing.  I think of the term "partner" as referring to how we like to work with them (our relationship), whereas "client" refers to our business arrangement.  But clearly since we disagree here, everyone does not share that understanding, so getting some input would feel useful.  I would suggest not holding up the changes, though, since what's here is still so much better than what's up currently.

In the meantime, if we don't like having both "client" and "partner" in the same sentence, how about changing "government partner" to "government agency"?

Reply:
Agree this could use a writing lap pass for consistency but for now I updated to: `you may find yourself in **partner-facing** consulting role to help a **government agency** build or buy`

Reply:
Small typo: "you may find yourself in **a** partner-facing ..."

Reply:
✔️